data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
 
 
 
+mission "Terraforming 13"
+	minor
+	name "Field Trip"
+	description "Amy, who you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
+	source "Tundra"
+	destination "Desi Seledrak"
+	passengers 1
+	to offer
+		has "event: terraforming follow-up"
+		has "wanderers sestor done"
+	on offer
+		conversation
+			`As you walk through the spaceport, you notice a poster featuring Amy. You previously helped her terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with how that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					decline
+			`	You're able to contact Amy. She has been continuing to monitor the status of Tundra's climate, alongside the researchers from the Deep who she worked with to terraform this world. You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them and agrees to meet you in the spaceport bar.`
+			`	Once she arrives, you explain about the Wanderers: that the work of their civilisation is to repair worlds left in ruin by their previous inhabitants. And their view that those worlds have their own identities that they must collaborate with them in order for terraforming to be successful.`
```

Comment:
As with roadrunner's suggestion, I don't like hte sound of the double "you explain".

Reply:
Right... I missed that first "you explain." That's on me. But in general, while I abide by the three sentence a paragraph rule religiously, here, it might be best to merge both sentences with the "and."